packages/entity-browser/src/routes/detail/components/DetailForm/DetailForm.js

```diff
@@ -150,6 +132,7 @@ DetailForm.propTypes = {
   formDefinition: React.PropTypes.object.isRequired,
   entity: React.PropTypes.object.isRequired,
   loadRelationEntity: React.PropTypes.func.isRequired,
+  formValues: React.PropTypes.object,
```

Comment:
its an object with any keys (fieldnames) and an unknown value type ( could be string, array ,object...)